docs/design/mono/web/ascii-strings.md

```diff
@@ -0,0 +1,137 @@
+---
+title: ASCII Mono
+---
+
+## Introduction
+
+This is a proposal for an optimisation to `System.String`.
```

Comment:
This describes abandoned proposal that was never productized. I would omit this doc.

We have https://github.com/dotnet/runtime/issues/6612 about this idea that discusses all sorts of problems that would have to be solved to make this idea productized.